packages/core/admin/admin/src/pages/Settings/pages/Users/EditPage.tsx

```diff
@@ -183,9 +183,14 @@ const EditPage = () => {
   const handleSubmit = async (body: InitialData, actions: FormikHelpers<InitialData>) => {
     lockApp?.();
 
+    const { confirmPassword, password, ...bodyRest } = body;
+
     const res = await updateUser({
       id,
-      ...omit(body, 'confirmPassword'),
+      ...bodyRest,
+      // The password should not be sent if it wasn't changed,
+      // it leads to a validation error if the string is empty
+      ...(password !== '' && { password }),
```

Comment:
Is there an issue in sending `null` or `undefined` instead? I think that would be a bit clearer.

Reply:
It will actually send `null` or `undefined` here, we only filter strings to avoid the validation error.

Reply:
The cleaner version may be to not make password "" by default and let it be undefined, which will avoid the need for checking anything here.

Reply:
`The cleaner version may be to not make password "" by default and let it be undefined` we use a Form for this page, can a form have `undefined` as it's default value without it being considered swapping from uncontrolled to controlled?

Reply:
Yep, you are right, it would lead to it changing from uncontrolled to controlled. If you have any suggestions on how I can improve it I would be happy to!

Reply:
```suggestion
      password: password === "" ? undefined : password,
```

I think this is probably fine?

Reply:
Yep, but I used the code that I used because there is a similar thing going on on the Profile Page (code that avoids sending `""` as a password) and it doesn't include password in the request at all, so I wanted to be consistent with what was already done:

```ts
if (dataToSend.password === '') {
  // ...
}
```

And the comment above it says:
```
// The password fields are optional. If the user didn't touch them, don't send any password
// to the API, because an empty string would throw a validation error
```

I guess I focused too much on the "DON'T SEND ANY PASSWORD" part.
  
<br />
But I have no particular preference. Feel free to go with any of the solutions 🙂